packages/wallet/src/mock-rollup-state-machine.ts

```diff
@@ -56,22 +56,24 @@ export class InvalidTransactionTypeError extends Error {
 export class MockRollupStateMachine {
   public state: State
 
-  constructor(genesisState: State) {
+  constructor(genesisState: State, private swapFeeBasisPoints: number = 30) {
     this.state = genesisState
   }
 
   public getBalances(account: Address): Balances {
     if (!(account in this.state)) {
-      return {
-        uni: 0,
-        pigi: 0,
+      this.state[account] = {
+        balances: {
+          uni: 0,
+          pigi: 0,
+        },
```

Comment:
Ah meant to comment on this in my PR review, but one thing that we don't want to do is store anything in the state based on a `getBalances()` query. Basically because that means that someone could technically bloat our state by simply asking for random balances they know are zero. That's why I simply returned the value

Reply:
Ahh ok. I can protect it from cases like that. Will update 👍

Reply:
Ok, this should be fixed now.

Reply:
Woot!